brave/brave-core
# Add filecoin testnet support

Resolves https://github.com/brave/brave-browser/issues/23368
 Resolves https://github.com/brave/brave-browser/issues/23241
 
Add separate keyring id for filecoin.
Add ui tweaks to support new keyring.


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1) Create filecoin account, create testnet account.
2) Check that created accs have different addresses.
3) Check that transactions work both on calibration and mainnet.
4) Try import mainnet and testnet accounts.
5) Check case when no accounts are for some filecoin network and this network is selected.
6) Try import hardware mainnet and testnet accounts.
7) Do smoke testing for other coins.
